test-data/unit/python2eval.test

```diff
@@ -189,7 +189,6 @@ f.write(u'foo')
 f.write('bar')
 f.close()
 [out]
-_program.py:3: error: Argument 1 to "write" of "IO" has incompatible type "unicode"; expected "str"
```

Comment:
This seems like a regression. `f` should be opened as a binary file in Python 2.